mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -216,6 +216,49 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 				},
 			},
 
+			<% unless version == "ga" -%>
+			"standby_policy": {
+				Computed:    true,
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: `Standby policy for stopped and suspended instances.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"initial_delay_sec": {
+							Type:             schema.TypeInt,
+							Default:          0,
+							Optional:         true,
+							ValidateFunc:     validation.IntBetween(0, 3600),
+							Description:      `Specifies the number of seconds that the MIG should wait to suspend or stop a VM after that VM was created. The initial delay gives the initialization script the time to prepare your VM for a quick scale out. The value of initial delay must be between 0 and 3600 seconds. The default value is 0.`,
+						},
+
+						"mode": {
+							Type:         schema.TypeString,
+							Default:      "MANUAL",
```

Comment:
If no value is provided in the API request for this field, will the API side default it to `MANUAL` and send it back in the API response?

Reply:
Same as the above. It will be default only during creation.